README.md

```diff
@@ -1,4 +1,10 @@
 # characters
```

Comment:
Remove superfluous title in readme. Redundant on both github page and pub site

Reply:
Done.